### PR TITLE
Update api_gpio.js

### PR DESCRIPTION
**Pull request by amizer12**
Externalise additional function hat allows to read the state of output GPIO pins

### CODE REVIEW

File: fs/api_gpio.js [hunk 1]
```diff
@@ -36,7 +36,10 @@ let GPIO = {
   // ## **`GPIO.read(pin)`**
   // Read GPIO pin level. Return value: 0 or 1.
   read: ffi('int mgos_gpio_read(int)'),
-
+  
+  //read GPIO level for OUTPUT GPIO mode. Return value: 0 or 1.
+  readout: ffi ('int mgos_gpio_read_out(int)'),
```
Comment (rojer) [inline fs/api_gpio.js:41]: read_out, for consistency with other names (even though really it should be readOut, setupOutput, but that's for another time)

Comment (rojer) [reply fs/api_gpio.js:41]: also please format the comment similar to other fiunctions.

Comment (amizer12) [reply fs/api_gpio.js]: Corrected

**Comment by rojer** [comment]
thank you!

**Pull request merged**